appcli/variables_manager.py

```diff
@@ -28,13 +31,15 @@
 class VariablesManager:
     """Manages the configuration variables"""
 
-    def __init__(self, configuration_file):
+    def __init__(self, configuration_file, extra_configuration_files):
         """Creates a manager for the specified file
 
         Args:
-            configuration_file (str): Path to the configuration file to manage
+            configuration_file (str): Path to the configuration file to manage.
+            extra_configuration_files (str): Path to a directory containing extra configuration files.
```

Comment:
`extra_configuration_files` is actually a str representation of a directory, but the name implies it may be an array

Reply:
Fixed.